sys-libs/glibc/glibc-2.35-r8.ebuild

```diff
@@ -818,12 +823,7 @@ upgrade_warning() {
 
 # pkg_pretend
 
-# Flatcar: Skip sanity checks at pretend time because we don't ship a compiler
-#  in the OS image. This test fails when installing the glibc binpkg and no
-#  compiler is present.
 pkg_pretend() {
-	einfo "Flatcar: Skipping sanity_prechecks for binpkg installation. src_unpack will take care of compile-time prechecks."
-	# sanity_prechecks
 	upgrade_warning
 }
```

Comment:
This change is not needed any more: https://github.com/gentoo/gentoo/blob/master/sys-libs/glibc/glibc-2.35-r8.ebuild#L826-L828